tests/integration/ceph_multi_cluster_test.go

```diff
@@ -52,6 +52,7 @@ const (
 // *************************************************************
 func TestCephMultiClusterDeploySuite(t *testing.T) {
 	if installer.SkipTestSuite(installer.CephTestSuite) {
+		logger.Infof("Skipping Ceph Multi Cluster Deploy Suite")
```

Comment:
there should already be a message logged inside `SkipTestSuite()`

Reply:
Done.